src/utils/transaction.js

```diff
@@ -134,52 +134,35 @@ function transferInt() {
     var oldBytes = convert_1.default.stringToByteArray('40004');
     var newBytes = oldBytes
     for (var key in newBytes) {
-        newBytes[key] = (oldBytes[key]-48).toString(16)
+        newBytes[key] = (oldBytes[key]-48)
     }
     return newBytes
 }
 function transferAmount(amountData) {
-    var oldBytes = convert_1.default.longToByteArray(amountData)
-    var newBytes = oldBytes
-    var typeArr = []
-
-    typeArr[0] = (3).toString(16)
-
-    for (var key in newBytes) {
-        newBytes[key] = (oldBytes[key]).toString(16)
-    }
-    return typeArr.concat(newBytes)
+    var byteArr = convert_1.default.longToByteArray(amountData)
```

Comment:
Do not use `longToByteArray`. This will cause precise loss issue. Please use `bigNumberToByteArray` instead.